rest/taskana-rest-spring-example-boot/src/main/resources/application-oracle.properties

```diff
@@ -0,0 +1,61 @@
+logging.level.pro.taskana=INFO
+server.servlet.context-path=/taskana
+### logging.level.org.springframework=DEBUG
+######## Taskana DB #######
+taskana.schemaName=taskana
+######## ORACLE configuration ########
+spring.datasource.url=jdbc:oracle:thin:@//localhost:5103/TASKANA
+spring.datasource.driverClassName=org.postgresql.Driver
+spring.datasource.username=TEST_USER
+spring.datasource.password=testPassword
+########spring.jpa.generate-ddl=true
+########spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+####### property that control rest api security deploy use true for no security.
+devMode=false
+####### property that control if the database is cleaned and sample data is generated
+generateSampleData=true
+####### properties to connect to LDAP
+taskana.ldap.serverUrl=ldap://localhost:10389
+taskana.ldap.bindDn=uid=admin,ou=system
+taskana.ldap.bindPassword=secret
+taskana.ldap.baseDn=o=TaskanaTest
+####### properties that control search for users and groups
+taskana.ldap.userSearchBase=ou=people
+taskana.ldap.userSearchFilterName=objectclass
+taskana.ldap.userSearchFilterValue=person
+taskana.ldap.userFirstnameAttribute=givenName
+taskana.ldap.userLastnameAttribute=sn
+taskana.ldap.userFullnameAttribute=cn
+taskana.ldap.userPhoneAttribute=phoneNumber
+taskana.ldap.userMobilePhoneAttribute=mobileNumber
+taskana.ldap.userEmailAttribute=email
+taskana.ldap.userOrglevel1Attribute=orgLevel1
+taskana.ldap.userOrglevel2Attribute=orgLevel2
+taskana.ldap.userOrglevel3Attribute=someDepartement
+taskana.ldap.userOrglevel4Attribute=orgLevel4
+taskana.ldap.userIdAttribute=uid
+taskana.ldap.groupSearchBase=ou=groups
+taskana.ldap.groupSearchFilterName=objectclass
+taskana.ldap.groupSearchFilterValue=groupOfUniqueNames
+taskana.ldap.groupNameAttribute=cn
+taskana.ldap.minSearchForLength=3
+taskana.ldap.maxNumberOfReturnedAccessIds=50
+taskana.ldap.groupsOfUser=memberUid
+####### JobScheduler cron expression that specifies when the JobSchedler runs
+taskana.jobscheduler.async.cron=0 * * * * *
+####### cache static resources properties
+spring.web.resources.cache.cachecontrol.cache-private=true
+####### for upload of big workbasket- or classification-files
+spring.servlet.multipart.max-file-size=10MB
+spring.servlet.multipart.max-request-size=10MB
+####### serve compressed files for faster UI loading times
+server.compression.enabled=true
+server.compression.mime-types=application/json,application/xml,text/html,text/xml,text/plain,application/javascript,text/css,image/svg+xml
+server.compression.min-response-size=10240
+spring.main.allow-bean-definition-overriding=true
+server.tomcat.max-http-post-size=-1
+server.tomcat.max-save-post-size=-1
+server.tomcat.max-swallow-size=-1
+####### tomcat is not detecting the x-forward headers from bluemix as a trustworthy proxy
+server.tomcat.remoteip.internal-proxies=.*
+server.forward-headers-strategy=native
```

Comment:
this is too big. Please reduce to datasource only. 
The spring-boot PR has done this adjustment. That's why you did it this way

Reply:
Nevermind. My fault. This is a new ticket. spring profile properties should only contain delta values.. If you're bored: Go ahead ^^ (in a new PR)

Reply:
done